std/assembly/rt/pure.ts

```diff
@@ -63,7 +63,7 @@ import { onincrement, ondecrement, onfree, onalloc } from "./rtrace";
 
 // @ts-ignore: decorator
 @global @unsafe
-function __visit(ref: usize, cookie: i32): void {
+export function __visit(ref: usize, cookie: i32): void {
```

Comment:
This one isn't useful externally

Reply:
Should I remove declaration from `rt/index.d.ts` as well?

Reply:
No, internally it is useful (used when implementing RT, which references that file), just not as part of the exported runtime interface.

Reply:
got it. Done!